### ISSUE TITLE
Enhancement - choose which controllers are pad 1-4 (and make the x360ce app support it)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [GivePass/x360ce#313]
```
Currently, the x360ce app relies on the order in which controls are installed 
to determine the order of controllers, so modifying the order of the guid's in 
the ini does not reflect ingame and in the x360ce app.

Preferably, the emulator would represent the controllers based on the order of 
the guid's as configured by the app.

Currently if you install a brand new control and configure it as controller 1, 
if you plug the old controller in =EVEN AFTER DELETING THE CURRENT INI= the 
pre-existing driver is assumed to be controller 1 and the brand new control is 
set as controller 2 even if the new control was controller 1 prior to plugging 
in the older. (I have verified that the preferred dinput device as set in 
joystick.cpl has no affect here)

Basically, what im saying is the pad/guid ordering should be persisent.

This would also technically require a GUI control to set the controls to their 
preferred pad number.
```

Original issue reported on code.google.com by `danialho...@gmail.com` on 15 Jun 2012 at 11:19
